Python/taskqueues-backup-images/main.py

```diff
@@ -31,8 +31,7 @@
 BACKUP_START_DATE = datetime(1995, 6, 17)
 BACKUP_COUNT = params.IntParam("BACKUP_COUNT", default=100).value()
 HOURLY_BATCH_SIZE = params.IntParam("HOURLY_BATCH_SIZE", default=600).value()
-BACKUP_BUCKET = params.StringParam("BACKUP_BUCKET",
-                                   input=params.ResourceInput).value()
+BACKUP_BUCKET = params.StringParam("BACKUP_BUCKET", input=params.ResourceInput).value()
```

Comment:
:(

Reply:
I can shorten the line length to prevent this. Will give it a go

Reply:
looks like it wrapped it, but differently. LGTY?